.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
They are helpful for contributors, otherwise contributors have to manually figure out how to set up the debugger (this can be especially difficult for TS-based projects to know what the correct parameters are). It would be preferable to leave them in, there's nothing specific to my workspace that's in the files.